config/locales/server.en.yml

```
@@ -1823,6 +1823,10 @@ en:
 
     ignore_user_enabled: "[Beta] Allow ignoring users."
 
+    ignored_users_count_message_threshold: "[Beta] Count of user ignores before sending a moderator notification."
+
+    ignored_users_message_gap_days: "[Beta] Number of gap days between sending user ignores moderator notifications."
```

```suggestion
    ignored_users_message_gap_days: "How long wait before notifying moderators again about a user who has been ignored by many others"
```

(This one is super hard to describe, I did my best.)